examples/with-stomp/withStomp.js

```diff
@@ -0,0 +1,39 @@
+import React, { useEffect, useState } from 'react'
+import { Stomp } from '@stomp/stompjs'
+
+const withStomp = (url, topic, headers = {}) => WrappedComponent => {
+  return props => {
+    const [message, setMessage] = useState({})
+    // connect to a stomp server over Websocket
+    const client = Stomp.client(url)
```

Comment:
Creating a client for every hook usage doesn't look right.

Reply:
You right, I will fix it.